manifests/noderesourcetopology/scheduler-config.yaml

```diff
@@ -18,8 +15,6 @@ profiles:
   pluginConfig:
   - name: NodeResourceTopologyMatch
     args:
-      # kubeconfig: "REPLACE_ME_WITH_KUBE_CONFIG_PATH"
-      kubeconfigpath: "/etc/kubernetes/scheduler.conf"
```

Comment:
I have one PR opened tracking this already: #290.

Reply:
Ack, Thanks!